Objects/longobject.c

```diff
@@ -47,8 +54,8 @@ get_small_int(sdigit ival)
 static PyLongObject *
 maybe_small_long(PyLongObject *v)
 {
-    if (v && Py_ABS(Py_SIZE(v)) <= 1) {
-        sdigit ival = MEDIUM_VALUE(v);
+    if (v && IS_MEDIUM_VALUE(v)) {
+        long ival = medium_value(v);
```

Comment:
Any particular reason for the `sdigit` -> `long` type change in this line? `sdigit` seems to make sense, given that `get_small_int` expects an `sdigit`.

Reply:
I think the change made sense when there was a `MEDIUM_VALUE` macro. I'll change it back.